sqlmesh/core/model/definition.py

```diff
@@ -536,14 +538,11 @@ def convert_to_time_column(
     def update_schema(
         self,
         schema: MappingSchema,
-        default_schema: t.Optional[str] = None,
         default_catalog: t.Optional[str] = None,
     ) -> None:
         """Updates the schema for this model's dependencies based on the given mapping schema."""
         for dep in self.depends_on:
-            table = d.set_default_schema_and_catalog(
-                dep, default_schema=default_schema, default_catalog=default_catalog
-            )
+            table = d.set_default_catalog(dep, default_catalog=default_catalog)
```

Comment:
Why are we doing this? Should `self.depends_on` return fqn already?

Reply:
yea depends on should be fully normalized

Reply:
yep